#Error Handling
# def sum(n1, n2):
#     try:
#         return n1 / n2
#     except ZeroDivisionError as err:
#         print(err)

# print(sum(10, '2'))

# while True:
#     try:
#         age = int(input("What's your age?"))
#         10/age
#     except ValueError:
#         print('Please enter a number')
#         continue
#     except ZeroDivisionError:
#         print('Please enter age greater than zeroo')
#         break
#     else:
#         print("Thank you")
#     finally:
#         print('Finally block is Executed!')
#     print("Hellloo")